packages/components/src/dropdown/styles.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { space } from '../ui/utils/space';
+import type { DropdownContentWrapperProps } from './types';
+
+const padding = ( { paddingSize = 'small' }: DropdownContentWrapperProps ) => {
```

Comment:
We should probably move the default assignment of `paddingSize` directly to the component's props in `packages/components/src/dropdown/dropdown-content-wrapper.tsx`

Reply:
Makes sense 👍